<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPointCloudSimplification</class>
 <widget class="QDialog" name="DialogPointCloudSimplification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sampling Resolution</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2">
    <widget class="QCheckBox" name="checkBoxUniform">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QRadioButton" name="radioButtonDistanceThreshold">
     <property name="text">
      <string>Specify distance thresold</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>55</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QRadioButton" name="radioButtonExpectedPointNumber">
     <property name="text">
      <string>Specify expected point number</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="qureyButton">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Query the points that can be deleted</string>
       </property>
       <property name="text">
        <string>Qurey</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Delete the queried points</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEditExpectedPointNumber">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Uniform</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEditDistanceThreshold">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>58</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Average spacing(k=6):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEditAverageSpacing">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="buttonComputeAvgSpacing">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Compute</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>radioButtonExpectedPointNumber</tabstop>
  <tabstop>lineEditExpectedPointNumber</tabstop>
  <tabstop>radioButtonDistanceThreshold</tabstop>
  <tabstop>lineEditDistanceThreshold</tabstop>
  <tabstop>lineEditAverageSpacing</tabstop>
  <tabstop>buttonComputeAvgSpacing</tabstop>
  <tabstop>checkBoxUniform</tabstop>
  <tabstop>qureyButton</tabstop>
  <tabstop>applyButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
